journalbeat/magefile.go

```diff
@@ -188,7 +189,7 @@ func installLinuxS390X() error {
 	return installDependencies(libsystemdDevPkgName+":s390x", "s390x")
 }
 
-func installDependencies(pkg, arch string) error {
+func installDependencies(arch string, pkgs ...string) error {
```

Comment:
this PR changes order of parameters, each call to 'installDependencies' must be modified.

Reply:
I swapped the arch & pkg.